### DISCUSSION TITLE
Collapse menu to big logo when clicking on empty space

### DISCUSSION THREAD

**Discussion by Gyoshi** [ppy/osu#23499] [Ideas] [answered]
Pressing Esc while the Main Menu is expanded causes it to collapse back to the big logo. 

This is quite a nice feature and it would be cool if clicking on the background (empty space) also did the same.

https://github.com/ppy/osu/assets/41743877/7807392e-e13f-4258-a0a0-3a54835d246f

**Comment by peppy** [comment]
Actually sounds like quite a nice change, will give it a try.

**Reply by peppy** [reply, marked as answer]
#23501
